tests/acceptance/AdvancedSettingsAnonDataToggleCest.php

```diff
@@ -2,18 +2,24 @@
 
 namespace MailPoet\Test\Acceptance;
 
-class AdvancedSettingsAnonDataToggleCest {
+class AdvancedSettingsAnonymousDataToggleCest {
   function toggleAnonDataSetting(\AcceptanceTester $I) {
-    $I->wantTo('Confirm anon data settings can be toggled');
-    $noAnonData = '[data-automation-id="analytics-no"]';
-    $yesAnonData = '[data-automation-id="analytics-yes"]';
+    $I->wantTo('Confirm anonymous data settings can be toggled');
+    $noAnonymousData = '[data-automation-id="analytics-no"]';
+    $yesAnonymousData = '[data-automation-id="analytics-yes"]';
+    $submitButton = '[data-automation-id="settings-submit-button"]';
     $I->login();
     $I->amOnMailPoetPage('Settings');
     $I->click('[data-automation-id="settings-advanced-tab"]');
     $I->waitForText('Bounce email address');
-    $I->selectOption($noAnonData, 'No');
-    $I->seeOptionIsSelected($noAnonData, 'No');
-    $I->selectOption($yesAnonData, 'Yes');
-    $I->seeOptionIsSelected($yesAnonData, 'Yes');
+    $I->selectOption($noAnonymousData, 'No');
+    $I->click($submitButton);
+    $I->waitForElementClickable($submitButton);
+    $I->seeOptionIsSelected($noAnonymousData, 'No');
+    $I->reloadPage();
+    $I->selectOption($yesAnonymousData, 'Yes');
+    $I->click($submitButton);
+    $I->waitForElementClickable($submitButton);
+    $I->seeOptionIsSelected($yesAnonymousData, 'Yes');
```

Comment:
Soo... I meant more something like this:
```
$I->selectOption($noAnonymousData, 'No');
$I->click($submitButton);
$I->waitForText('Settings saved');
$I->reloadPage();
$I->seeOptionIsSelected($noAnonymousData, 'No');
```

Meaning that I see the option correctly selected after change & save & reload. I guess no need to reload between testing the two options.

Reply:
Saving refreshes the page, so I cut out the reload all together.